apps/core-app/src/Routes.tsx

```diff
@@ -14,31 +14,35 @@ import Settings from './pages/Settings';
 import NewProposal from './pages/NewProposal';
 import UpdateSettings from './pages/UpdateSettings';
 import ProposalDetails from './pages/ProposalDetails';
+import { Banner } from '@daohaus/ui';
 
 const Routes = () => {
   return (
-    <RoutesDom>
-      <Route path="/" element={<Home />} />
-      <Route
-        path="molochv3/:daochain/:daoid"
-        element={
-          <DaoContextProvider>
-            <Dao />
-          </DaoContextProvider>
-        }
-      >
-        <Route index element={<DaoOverview />} />
-        <Route path="formtest" element={<FormTest />} />
-        <Route path="proposals" element={<Proposals />} />
-        <Route path="new-proposal" element={<NewProposal />} />
-        <Route path="proposals/:proposalId" element={<ProposalDetails />} />
-        <Route path="vaults" element={<Vaults />} />
-        <Route path="members" element={<Members />} />
-        <Route path="members/:memberAddress" element={<Member />} />
-        <Route path="settings" element={<Settings />} />
-        <Route path="settings/update" element={<UpdateSettings />} />
-      </Route>
-    </RoutesDom>
+    <>
+      <Banner />
```

Comment:
I wonder if this would be better placed in https://github.com/HausDAO/daohaus-monorepo/blob/master/libs/daohaus-connect-feature/src/components/HausLayout/HausLayout.tsx?

if that layout is used in other apps?